.gitignore

```diff
@@ -131,6 +131,8 @@ dmypy.json
 
 # Test and spec generated
 reports/
+aviary/reports/
+*.openmdao_out
```

Comment:
Just want to make sure you know about the `openmdao clean` command, for cleaning up all of these directories.
https://openmdao.org/newdocs/versions/latest/other_useful_docs/om_command.html#openmdao-clean